nbviewer/app.py

```diff
@@ -281,6 +282,7 @@ def init_options():
     define("debug", default=False, help="run in debug mode", type=bool)
     define("no_cache", default=False, help="Do not cache results", type=bool)
     define("localfiles", default="", help="Allow to serve local files under /localfile/* this can be a security risk", type=str)
+    define("localfile_only_world_readable", default=False, help="Only serve files that are readable by 'Other' on the local file system", type=bool)
```

Comment:
In the interest of having the most secure defaults and options to loosen them up, maybe this should be `--localfile_any_user` or some similarly named inverse?

Reply:
Nice suggestion, I'm not great at naming stuff. ;-)

Will do.

Reply:
No worries. Naming *is* hard.